django_declarative_apis/machinery/__init__.py

```diff
@@ -101,9 +103,11 @@ def get_instance_value(self, owner_instance, owner_class):
         try:
             value = self.func(owner_instance)
         except django.core.exceptions.ObjectDoesNotExist:
-            raise errors.ClientErrorNotFound(
-                "{0} instance not found".format(self.type.__name__)
-            )
+            if hasattr(self.type, "__name__"):
+                message = "{0} instance not found".format(self.type.__name__)
```

Comment:
f-string would be preferred here, no? or are we concerned about preserving python backwards compatibility?